app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -353,13 +353,13 @@ def parse_storage_pod(object, kind, props)
     persister.ems_folders.targeted_scope << object._ref
     return if kind == "leave"
 
-    name = props.fetch_path(:summary, :name)
+    name = CGI.unescape(props[:name]) if props[:name]
```

Comment:
On the go vcsim there is no `props[:summary][:name]` but there is a `props[:name]`.  These are the same property and we can use `props[:name]` for live VCs as well.